### PR TITLE
[fix/PLAYER-5165] Airplay icon is not displayed as expected on refresh/reload

### PR DESCRIPTION
**Pull request by ALexanderSPB**
https://jira.corp.ooyala.com/browse/PLAYER-5165

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -500,12 +500,13 @@ module.exports = function(OO, _, $, W) {
     toggleAirPlayIcon: function(event) {
       if (!this.state.airPlayStatusIcon) {
         const airPlayState = window.sessionStorage.getItem('airPlayState');
-        if (airPlayState) {
-          this.state.airPlayStatusIcon = airPlayState;
-          this.state.mainVideoElement.webkitShowPlaybackTargetPicker();
-        } else {
-          this.state.airPlayStatusIcon = 'airPlay-disconnected';
+        if (airPlayState === 'airPlay-connected') {
```
Comment (ooyala-amohnatkin) [inline js/controller.js]: I guess we should keep `airPlay-connected` as a constant

File: js/controller.js [hunk 2]
```diff
@@ -500,12 +500,13 @@ module.exports = function(OO, _, $, W) {
...
+          setTimeout(() => {
```
Comment (ooyala-amohnatkin) [inline js/controller.js]: please add a comment why do you use setTimeout and why it's so big

**Comment by zelibobla** [comment]
Every PR should be followed by description what is done and why

File: js/controller.js (hunk 1 above)
Comment (ALexanderSPB) [reply js/controller.js]: Fixed

File: js/controller.js (hunk 2 above)
Comment (ALexanderSPB) [reply js/controller.js]: I thought it would be more userfriendly to show it with some delay after buffering. But it works without settimeout.

**Comment by ALexanderSPB** [comment]
When airPlayState === CONSTANTS.AIRPLAY_STATE.CONNECTED, 
Safari calls toggleAirPlayIcon 2 times  but when it is DISCONNECTED toggleAirPlayIcon is called 1 time. Thats why Airplay icon was not displayed as expected on refresh/reload. So I set this.state.airPlayStatusIcon to CONSTANTS.AIRPLAY_STATE.DISCONNECTED in any case.

**Pull request merged**